src/streamlink_cli/main.py

```diff
@@ -125,6 +129,16 @@ def create_http_server(host=None, port=0):
 
     return http
 
+def create_title(plugin=None):
+    if args.title and plugin:
+        title = LazyFormatter.format(maybe_encode(args.title),
```

Comment:
could you format it like

```py
def create_title(plugin=None):
    if args.title and plugin:
        title = LazyFormatter.format(
            maybe_encode(args.title),
            title=lambda: plugin.get_title() or DEFAULT_STREAM_METADATA["title"],
            author=lambda: plugin.get_author() or DEFAULT_STREAM_METADATA["author"],
            category=lambda: plugin.get_category() or DEFAULT_STREAM_METADATA["category"],
            game=lambda: plugin.get_category() or DEFAULT_STREAM_METADATA["game"]
        )
    else:
        title = args.url
    return title
```

`E128 continuation line under-indented for visual indent`